### ISSUE TITLE
80 character limit across whole site

### ISSUE THREAD

**Issue by ellemindwave** [LDMW/cms#728]
For accessibility reasons, no line of text on the site anywhere should be more than 80 characters

**Comment by Danwhy** [comment]
@ellemindwave @sarahwelton With the character limit applied, the paragraphs are now smaller than other elements on the page. Should the text still align with the edges of the other elements, or should it be centered?
<img width="803" alt="screen shot 2017-10-25 at 14 45 45" src="https://user-images.githubusercontent.com/8939909/32002094-858c81ea-b993-11e7-9a66-62b1325f67c5.png">

<img width="814" alt="screen shot 2017-10-25 at 14 47 24" src="https://user-images.githubusercontent.com/8939909/32002116-92d0c6ae-b993-11e7-82c1-2d6b96c854df.png">

**Comment by sarahwelton** [comment]
Aligned to the left edge please
